src/main/java/com/hyunjin/funding/Security/SecurityConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package com.hyunjin.funding.Security;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Slf4j
+@Configuration
+@EnableWebSecurity
+// api에 권한 어노테이션을 달아서 api 접근 권한 설정 가능하도록 함.
+@EnableGlobalMethodSecurity(prePostEnabled = true)
```

Comment:
```suggestion
@EnableGlobalMethodSecurity(prePostEnabled = true) // api에 권한 어노테이션을 달아서 api 접근 권한 설정 가능하도록 함.
```

// 형태의 로그는 통상 오른쪽에 달아줘요. (다른 로깅 부분은 💯 )

Reply:
주석이 너무 길어서 위로 올렸었는데 주석을 줄여서 오른쪽으로 옮겼습니다!